resources/Microsoft.DSC.Experimental/0.0.1/Microsoft.DSC.Experimental.psm1

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+using namespace System.Collections.Generic
+
+[DscResource()]
+class Service
+{
+    [DscProperty(Key)]
+    [string] $Unit
+
+    [DscProperty()]
+    [string] $Load
+
+    [DscProperty()]
+    [string] $Active
+
+    [DscProperty()]
+    [string] $Sub
+
+    [DscProperty()]
+    [string] $Description
+
+    [void] Set()
+    {
+    }
+
+    [bool] Test()
+    {
+        return $false
+    }
+
+    [Service] Get()
+    {
+       
+        return [Service]::GetServices() | ? {$_.Unit -eq $this.Unit }
+    }
+
+    static [Service[]] Export()
+    {
+        $resultList = [List[Service]]::new()
+        $svcs = [Service]::GetServices()
+        $svcs | %{
+            $obj = New-Object Service
+            $obj.Unit = $_.Unit
+            $obj.Load = $_.Load
+            $obj.Active = $_.Active
+            $obj.Sub = $_.Sub
+            $obj.Description = $_.Description
+            
+            $resultList.Add($obj)
+        }
+
+        return $resultList.ToArray()
+    }
+
+    static [pscustomobject[]] GetServices()
```

Comment:
Is there a reason we return an array of custom objects instead of instances of the class? I also think it might be useful to just shorten the method name to `Get()` or `GetAll()` since the class name already indicates that it's going to retrieve services.

Reply:
Changed to `[SystemctlService[]] GetAll()`.